app/assets/stylesheets/blacklight.scss

```diff
@@ -1,3 +1,16 @@
 @import 'bootstrap';
 
 @import 'blacklight/blacklight';
+
+.blacklight-icons svg {
```

Comment:
Were these removed upstream?

Reply:
Yes. See https://github.com/projectblacklight/blacklight/pull/2225 and https://github.com/projectblacklight/blacklight/issues/2230